tests/components/calendar/test_init.py

```diff
@@ -71,6 +71,23 @@ async def test_events_http_api_error(
         assert await response.json() == {"message": "Error reading events: Failure"}
 
 
+async def test_events_http_api_dates_wrong_order(
+    hass: HomeAssistant, hass_client: ClientSessionGenerator
+) -> None:
+    """Test the calendar demo view."""
+    await async_setup_component(hass, "calendar", {"calendar": {"platform": "demo"}})
+    await hass.async_block_till_done()
+    client = await hass_client()
+    start = dt_util.now()
+    end = start + timedelta(days=-1)
+    response = await client.get(
+        "/api/calendars/calendar.calendar_1?start={}&end={}".format(
+            start.isoformat(), end.isoformat()
+        )
```

Comment:
Please use f-strings instead of format()

Reply:
Okey, should I change the other calendar tests as well? I just used the same as the others.